reader_test.go

```diff
@@ -641,6 +641,30 @@ func TestStrictTimeParse(t *testing.T) {
 	}
 }
 
+func TestMediaPlaylistWithSimpleCues(t *testing.T) {
+	f, err := os.Open("sample-playlists/media-playlist-with-simple-cues.m3u8")
+	if err != nil {
+		t.Fatal(err)
+	}
+	p, _, err := DecodeFrom(bufio.NewReader(f), true)
+	if err != nil {
+		t.Fatal(err)
+	}
+	pp := p.(*MediaPlaylist)
+
+	expect := map[int]*SCTE{
+		0: {Syntax: SCTE35_SIMPLECUES, CueType: SCTE35Cue_Start, Time: 15},
+		2: {Syntax: SCTE35_OATCLS, CueType: SCTE35Cue_End},
```

Comment:
we are already reading `#EXT-X-CUE-IN` tag as a `SCTE35_OATCLS`. I decided to leave as is. We don't necessarily do anything with syntax type outside of writing to the manifest.